README.md

```diff
@@ -1,89 +1,74 @@
-|------------|-----|-----|------------|-----------------|--------|-------------|
-| Caffe      | Y   | Y   | Y          | Y               | Y      |Y            |
-| Caffe2     | N   | N   | N          | N               | Y      |N            |
-| XGBoost    | Y   | Y   | Y          | N               | N      |N            |
-| Tensorflow | N   | N   | N          | N               | Y      |N            |
-| T-SNE      | Y   | N   | N          | Y               | Y      |N            |
-| Dlib       | N   | N   | N          | N               | Y      |N            |
-| TensorRT   | N   | N   | N          | N               | Y      |N            |
-| Libtorch   | N   | N   | N          | N               | Y      |N            |
-
-#### Main functionalities
-
-DeepDetect implements support for supervised and unsupervised deep learning of images, text, time series and other data, with focus on simplicity and ease of use, test and connection into existing applications. It supports classification, object detection, segmentation, regression, autoencoders, ...
-
-#### Support
-
-Please join either the community on [Gitter](https://gitter.im/beniz/deepdetect) or on IRC Freenode #deepdetect, where we help users get through with installation, API, neural nets and connection to external applications.
-
-#### Supported Platforms
-
-The reference platforms with support are **Ubuntu 16.04 LTS** and **Ubuntu 18.04 LTS**.
-
-Supported images that come with pre-trained image classification deep (residual) neural nets:
-
-- **docker images** for CPU and GPU machines are available at https://hub.docker.com/u/jolibrain. See https://github.com/jolibrain/deepdetect/tree/master/docker/README.md for details on how to use them.
-
-- For **Amazon AMI** see links for [GPU AMI](https://aws.amazon.com/marketplace/pp/B01N4D483M) and [CPU AMI](https://aws.amazon.com/marketplace/pp/B01N1RGWQZ).
-
-#### Performances
-
-See https://github.com/jolibrain/dd_performances for a report on performances on NVidia Desktop and embedded GPUs, along with Raspberry Pi 3.
-
-#### Quickstart
-Setup an image classifier API service in a few minutes:
-https://www.deepdetect.com/server/docs/imagenet-classifier/
-
-#### Tutorials
-List of tutorials, training from text, data and images, setup of prediction services, and export to external software (e.g. ElasticSearch): https://www.deepdetect.com/server/docs/server_docs/
-
-#### Features and Documentation
-Current features include:
+Please join either the community on [Gitter](https://gitter.im/beniz/deepdetect), where we help users get through with installation, API, neural nets and connection to external applications.
+
+---
+
+* [Machine Learning functionalities per library](#machine-learning-functionalities-per-library)
+* [Main features](#main-features)
+* [Installation](#installation)
+  * [From docker](#installation-from-docker)
+  * [From source](#installation-from-source)
```

Comment:
We may want to directly link to the `docs/source.md` file ? Same for docker, etc...